internal/services/logz/logz_sub_account_tag_rule_resource.go

```diff
@@ -0,0 +1,154 @@
+package logz
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/logz/mgmt/2020-10-01/logz"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/tf"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/logz/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/logz/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/timeouts"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+func resourceLogzSubAccountTagRule() *pluginsdk.Resource {
+	return &pluginsdk.Resource{
+		Create: resourceLogzSubAccountTagRuleCreateUpdate,
+		Read:   resourceLogzSubAccountTagRuleRead,
+		Update: resourceLogzSubAccountTagRuleCreateUpdate,
+		Delete: resourceLogzSubAccountTagRuleDelete,
+
+		Timeouts: &pluginsdk.ResourceTimeout{
+			Create: pluginsdk.DefaultTimeout(30 * time.Minute),
+			Read:   pluginsdk.DefaultTimeout(5 * time.Minute),
+			Update: pluginsdk.DefaultTimeout(30 * time.Minute),
+			Delete: pluginsdk.DefaultTimeout(30 * time.Minute),
+		},
+
+		Importer: pluginsdk.ImporterValidatingResourceId(func(id string) error {
+			_, err := parse.LogzSubAccountTagRuleID(id)
+			return err
+		}),
+
+		Schema: map[string]*pluginsdk.Schema{
+			"logz_sub_account_id": {
+				Type:         pluginsdk.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.LogzSubAccountID,
+			},
+
+			"tag_filter": schemaTagFilter(),
+
+			"send_aad_logs": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+
+			"send_activity_logs": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+
+			"send_subscription_logs": {
```

Comment:
should these be
```suggestion
			"send_aad_logs_enabled": {
				Type:     pluginsdk.TypeBool,
				Optional: true,
				Default:  false,
			},

			"send_activity_logs_enabled": {
				Type:     pluginsdk.TypeBool,
				Optional: true,
				Default:  false,
			},

			"send_subscription_logs_enabled": {
```

Reply:
Hi @katbyte, I totally agree with this comment. But since these fields are shared by azurerm_logz_tag_rule and azurerm_logz_sub_account_tag_rule, and azurerm_logz_tag_rule has already been released, if I changed the field names here, it would cause a breaking change for azurerm_logz_tag_rule resource. So, is it possible to keep them unchanged this time?